styles/light/_light-theme.scss

```diff
@@ -643,7 +643,20 @@ pre code {
   #wallet-xumm {
     content: url("../img/wallets/lightmode/xumm.svg");
   }
+  #wallet-bitfrost {
+    content: url("../img/wallets/lightmode/bitfrost.png");
+  }
+  #wallet-towo {
+    content: url("../img/wallets/lightmode/towo.svg");
+  }
+}
+#wallet-keystone {
+  content: url("../img/wallets/lightmode/keystone.svg");
 }
+#wallet-dcent {
+  content: url("../img/wallets/lightmode/dcent.svg");
+}
```

Comment:
This section does not need to be here if you're using `invertible-img` for d'cent.